docs/data-sources/gateway_control_plane.md

```diff
@@ -17,17 +17,17 @@ data "konnect_gateway_control_plane" "my_gatewaycontrolplane" {
   filter = {
     cloud_gateway = true
     cluster_type = {
-      eq  = "...my_eq..."
-      neq = "...my_neq..."
+      eq  = "?filter[field_name_here][eq]=foo"
+      neq = "?filter[field_name_here][neq]=bar"
```

Comment:
This is the `example` field from the spec, but it doesn't make sense as an example in Terraform. Can we fix all filter schema examples with overlays?